selection-to-clipboard/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "manifest_version": 2,
+    "name": "selection-to-clipboard",
+    "description": "Example of WebExtensionAPI for writing to the [clipboard](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Interact_with_the_clipboard) ",
+    "version": "1.0",
+    "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/selection-to-clipboard",
+
+    "icons": {
+        "48": "icons/clipboard-48.png"
+    },
+
+    "applications": {
+        "gecko": {
+            "id": "selection-to-clipboard@mozilla.org",
+            "strict_min_version": "49.0a1"
```

Comment:
I think this was fixed in https://bugzilla.mozilla.org/show_bug.cgi?id=1197451, which landed in Firefox 51. I tried this in nightly and it worked, but it didn't in release, Firefox 49. So I think we should increase this strict_min_version to a working Firefox version.